referencesrv/resolver/common.py

```diff
@@ -64,7 +64,10 @@ def __lt__(self, other):
         if hasattr(other, 'get_score'):
             return self.get_score()<other.get_score()
         else:
-            return self.get_score()<other
+            try:
```

Comment:
here I'd really like to protest against the wide try/except -- since the original code propagated exceptions, i'd think the new one can as well

can the wide except be limited to only some Exceptions?

Reply:
if other cannot be cast to int is the only time the exception happens. added `ValueError`.